### DISCUSSION TITLE
Point a domain from Netlify to Vercel

### DISCUSSION THREAD

**Discussion by zvitez** [vercel/community#4182] [Help]
### Summary

I have a domain registered on Netlify I'd like to point to a Vercel app. It's currently set up as:
`Name:
startupguide.com
TTL:
3600 seconds
Type:
NETLIFY
Value:
startupguide.netlify.app`
To point it to Vercel, do I just need to remove this NETLIFY record, and add a CNAME to one provided by Vercel?

### Steps to Reproduce

*

**Comment by amyegan** [comment]
Yes, you can point the domain away from Netlify to your Vercel project using DNS records. 🙂 

General instructions for configuring a custom domain can be found here: [Adding & Configuring a Custom Domain](https://vercel.com/docs/projects/domains/add-a-domain#)
